index.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="node"/>
```

Comment:
This causes users to get a more informative error message than:

> error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.

See https://github.com/sindresorhus/typescript-definition-style-guide#checklist